src/schemathesis/_hypothesis.py

```diff
@@ -33,15 +36,22 @@ def create_test(
 ) -> Callable:
     """Create a Hypothesis test."""
     hook_dispatcher = getattr(test, "_schemathesis_hooks", None)
-    strategy = endpoint.as_strategy(hooks=hook_dispatcher)
+    from .runner.impl.core import Feedback  # pylint: disable=import-outside-toplevel
+
+    feedback = Feedback("links", endpoint)
```

Comment:
The "stateful" parameter should be "links", created feedback only in this case. Or maybe use enum already?

Reply:
If so, then that enum should be used in the runner as well